PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,8 @@
+#### What's this PR do?
+#### Who is the reviewing it?
```

Comment:
I think there's a typo here? Should be `Who is the <noun> reviewing it` or `Who is reviewing it`.